### ISSUE TITLE
[Feature-Request] Telegram: Only receive notifications for reservations 

### ISSUE THREAD

**Issue by ddb9797** [Der-Henning/tgtg#482]
Is there a way to only receive notifications for made reservations?

**Comment by floriegl** [comment]
Yes, there is an unofficial hack. You could misconfigure the TELEGRAM_BODY to not be valid Markdown V2 which raises a BadRequest for all item updates except reservations (by using two stars after each other `'TELEGRAM_BODY=*$${{display_name}}**`). https://github.com/Der-Henning/tgtg/blob/main/tgtg_scanner/notifiers/telegram.py#L241
